task_schedules/task_schedule_update_supplement_rc.cfg

```diff
@@ -11,17 +11,28 @@ loud              0                 # Run loudly or quietly (production mode)
 # Data files and directories.  The *_dir vars can have $ENV{} vars which
 # get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
 
-data_dir     $ENV{SKA}/data/agasc/rc/tasks/update       # Data file directory
+data_dir     $ENV{SKA}/data/agasc/rc/tasks       # Data file directory
 log_dir      $ENV{SKA}/data/agasc/logs  # Log file directory
+heartbeat    update_heartbeat
 #bin_dir     ///     # Bin dir (optional, see task def'n)
 master_log   agasc_supplement_update.log             # Composite master log (created in log_dir)
 
 # Email addresses that receive an alert if there was a severe error in
 # running jobs (i.e. couldn't start jobs or couldn't open log file).
 # Processing errors *within* the jobs are caught with watch_cron_logs
 
-alert        javier.gonzalez@cfa.harvard.edu
-notify       javier.gonzalez@cfa.harvard.edu
```

Comment:
I assume this still needs a notify if the notify_msg below is to do anything?

Reply:
I don't understand what you mean. That is because I have not used this yet, but I got the impression this should work already.

Reply:
When would it send the notify_msg?

Reply:
this change did not remove the notify/alert lines. It replaced them. Now it sends to aca@cfa instead of javier.gonzalez@cfa (next two lines in the diff).

Reply:
ah, I see. Weird.

Reply:
In the file I commented on, at this location, it removes two lines and only adds one back.

Reply:
And now you've updated that.